packages/govuk-frontend/src/govuk/components/tag/_index.scss

```diff
@@ -1,12 +1,15 @@
 @include govuk-exports("govuk/component/tag") {
   .govuk-tag {
-    @include govuk-font($size: 16, $weight: bold, $line-height: 1);
+    @include govuk-font($size: 19);
 
     display: inline-block;
 
-    padding-top: 5px;
+    margin-top: -3px;
```

Comment:
These margins are intended to be global so that using a tag doesn’t increase the height of a parent container. For example, if you have a table row with a tag, and one without, their heights will be different.

Reply:
Example:

![tag-spacing](https://github.com/alphagov/govuk-frontend/assets/813383/5cfc9d32-b981-4cee-98b6-8827da08b576)

Maybe we should add a comment to explain these negative margin values?

Reply:
Added a comment in 68683bc – @paulrobertlloyd would you mind double-checking that the maths is right?

Reply:
Oh, actually the negative margins should match their equivalent positive padding values, meaning the top margin should be `-2px` (this doesn’t actually change the spacing, but it’s less clear why the value would be different).

Reply:
@frankieroberto Did you update this? /cc @owenatgov

Reply:
@paulrobertlloyd I can't actually remember, there’s been so many tweaks and changes! 

But making the negative margin match the padding makes sense!

@colinrotherham I'm not familiar with how the `govuk-em` mixin works. Should we use this for all the `px` values?

Reply:
Yes please to making them both match ✅

Regarding text scaling (e.g. Safari `⌥ ⌘ +` **Make Text Bigger**) I'm going to tag in @owenatgov

I'd like to say yes but I'm aware we have a [hard 10px spacing in `phase-banner`](https://github.com/frankieroberto/govuk-frontend/blob/552a49ad67308eb1feaea66eb32caef92d694dc1/packages/govuk-frontend/src/govuk/components/phase-banner/_index.scss#L5) too:

```css
padding-top: govuk-spacing(2);
padding-bottom: govuk-spacing(2);
```

So if we make the the tag itself "font-size scalable" the phase banner itself won't be

Reply:
@colinrotherham I've made the negative margin match the padding in 54390308dc46d64203da7d76015db4571adc4811 (and added a comment to explain).

Do you want me to replace all the static `<x>px` values with `govuk-em(<x>, 16)` or should that be left to a follow-up PR? @owenatgov